Source/PersonalityInsightsV3/Models/ContentItem.swift

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import RestKit
+
+//TODO - figure out where ContentListContainer object is in order to possibly pass in userid and sourceid (https://www.ibm.com/watson/developercloud/doc/personality-insights/relnotes.shtml)
```

Comment:
Ah yes! I actually just now finished my conversation with Watson's Personality Insight Slack channel asking about a documentation question regarding the removal of the `userid`, `sourceid` and `charset` as they had specified right after removing the parameters that "You pass these objects in the body of a JSON request as elements of the contentItems array of the ContentListContainer object.” 

I wanted to make sure I followed up with someone to make sure the parameters were removed completely and I needn't re-add the parameters they wanted removed. They got back to me today with the clarification to remove them.